### DISCUSSION TITLE
Is Mavic 3 compatible now?

### DISCUSSION THREAD

**Discussion by The1only** [RosettaDrone/rosettadrone#165] [General]
Has any one tested DJI Mavic 3?

**Comment by adpmdrones** [comment]
We've compiled RosettaDrone with DJI SDK v5
But we need some hints on the Google Play Seervices
![main_screen](https://github.com/RosettaDrone/rosettadrone/assets/4736951/c8baadab-614d-45b5-8a4e-4a51c0e88203)
![20231219_074024](https://github.com/RosettaDrone/rosettadrone/assets/4736951/ecafbcba-a20f-40ef-93be-011c5876bc6d)
